### PR TITLE
doc: correct rbd snap-clone doc

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: docs/design/proposals/rbd-snap-clone.md [hunk 1]
```diff
@@ -18,42 +18,42 @@
     - [RBD CLI commands to create a Volume from Volume](#rbd-cli-commands-to-create-a-volume-from-volume)
 
 This document outlines the command used to create RBD snapshot, delete RBD
-snapshot Restore RBD snapshot and Create New RBD image from RBD image.
+snapshot, Restore RBD snapshot and Create new RBD volume from existing RBD volume.
```
Comment (Madhu-1) [inline docs/design/proposals/rbd-snap-clone.md:21]: rbd volume should be `rbd image`  as RBD represents it's as an image

Comment (humblec) [reply docs/design/proposals/rbd-snap-clone.md]: sure. thanks . done @Madhu-1

**Comment by Madhu-1** [comment]
@mergifyIO rebase

**Comment by humblec** [comment]
@Madhu-1 why this is not getting merged?

**Comment by Madhu-1** [comment]
> @Madhu-1 why this is not getting merged?

few PR's are in waiting list https://github.com/ceph/ceph-csi/pull/1339/checks?check_run_id=969975786

**Comment by nixpanic** [comment]
@humblec could you manually rebase this and force-push the PR? Commitlint added a merge commit with the master branch, and that seems to cause issues with `/test commitlint`. Once re-pushed, other PRs should get merged again, and hopefully the GitHub commitlint App reports its status to this PR :crossed_fingers:

**Comment by humblec** [comment]
> @humblec could you manually rebase this and force-push the PR? Commitlint added a merge commit with the master branch, and that seems to cause issues with `/test commitlint`. Once re-pushed, other PRs should get merged again, and hopefully the GitHub commitlint App reports its status to this PR crossed_fingers

Done.  lets see :) @nixpanic

**Comment by humblec** [comment]
@Madhu-1 @nixpanic can you bring back approvals?

**Pull request merged**